### ISSUE TITLE
Uploading exercise error

### ISSUE THREAD

**Issue by tomurb** [ohtu-intellij/tmc-intellij#121]
Error occured when I was uploading test using TMC Idea. All tests passed when I used TMC NetBeans.

Error massage: org.apache.commons.lang.StringUtils.join(Ljava/util/Collection;Ljava/lang/String;)Ljava/lang/String;
```
org.apache.commons.lang.StringUtils.join(Ljava/util/Collection;Ljava/lang/String;)Ljava/lang/String;
java.lang.NoSuchMethodError: org.apache.commons.lang.StringUtils.join(Ljava/util/Collection;Ljava/lang/String;)Ljava/lang/String;
	at fi.helsinki.cs.tmc.intellij.ui.submissionresult.SuccessfulSubmissionDialog.getPointsMsg(SuccessfulSubmissionDialog.java:214)
	at fi.helsinki.cs.tmc.intellij.ui.submissionresult.SuccessfulSubmissionDialog.addPointsLabel(SuccessfulSubmissionDialog.java:200)
	at fi.helsinki.cs.tmc.intellij.ui.submissionresult.SuccessfulSubmissionDialog.<init>(SuccessfulSubmissionDialog.java:70)
	at fi.helsinki.cs.tmc.intellij.ui.submissionresult.SubmissionResultHandler.showResultMessage(SubmissionResultHandler.java:19)
	at fi.helsinki.cs.tmc.intellij.services.exercises.ExerciseUploadingService.getSubmissionResult(ExerciseUploadingService.java:108)
	at fi.helsinki.cs.tmc.intellij.services.exercises.ExerciseUploadingService.lambda$getResults$0(ExerciseUploadingService.java:84)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$1(CoreProgressManager.java:176)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:556)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:501)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:66)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:163)
	at fi.helsinki.cs.tmc.intellij.services.ThreadingService.lambda$runWithNotification$0(ThreadingService.java:20)
	at com.intellij.openapi.application.impl.ApplicationImpl$2.run(ApplicationImpl.java:334)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by anothermindbomb** [comment]
I have the same issue. Manually reverted back to 1.0.3 from the JetBrains repository and uploads work once again.

**Comment by LofhJann** [comment]
Fixed on my fork by changing to use org.apache.commons.lang3 instead of org.apache.commons.lang, seemingly fixed the problem. Waiting for merge.

**Comment by nygrenh** [comment]
Should be fixed in #122.
